modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotStorage.scala

```diff
@@ -43,25 +41,32 @@ object GlobalSnapshotStorage {
         }
       }.flatten
 
-    def get(ordinal: SnapshotOrdinal): F[Option[GlobalSnapshot]] =
+    def get(ordinal: SnapshotOrdinal): F[Option[Signed[GlobalSnapshot]]] =
       snapshotsRef.get.map(_.find(_.ordinal === ordinal))
 
-    def getLast: F[GlobalSnapshot] = snapshotsRef.get.map(_.head)
+    def getLast: F[Signed[GlobalSnapshot]] = snapshotsRef.get.map(_.head)
 
-    def getStateChannelSnapshotUntilOrdinal(
-      ordinal: SnapshotOrdinal
-    )(address: Address): F[Option[StateChannelSnapshotBinary]] =
+    /**
+      * Temporary implementation - until we agree on a representation and generation of the GlobalSnapshotInfo objects
```

Comment:
if it's going to stay in the codebase, can you mark it with `TODO: ` or `Note: `? it will be easier to track later

Reply:
Actually, I think I will remove this comment.